python/tests/test_everything.py

```diff
@@ -0,0 +1,2 @@
+def test_test():
+    pass
```

Comment:
todo: real unit tests go here

Reply:
@DanielCollins Did you want to add these tests in another PR?

Reply:
yes

Reply:
here i just demonstrate that testing works